### PR TITLE
add new chat members handler

### PR DESCRIPTION
**Pull request by imorte**

### CODE REVIEW

**Comment by imorte** [comment]
@vjgarciag96 Could you look, please?

**Comment by vjgarciag96** [comment]
Sure, I'll probably take a look to it tomorrow

File: telegram/src/main/kotlin/com/github/kotlintelegrambot/dispatcher/handlers/NewChatMembersHandler.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+class NewChatMembersHandler(
+    handlerCallback: HandleUpdate
+) : Handler(handlerCallback) {
+    override val groupIdentifier: String
+        get() = "System"
+
+    override fun checkUpdate(update: Update): Boolean {
+        return update.message?.newChatMembers != null
+    }
+}
```
[the remark this answers is not in the document]
Comment (imorte) [reply telegram/src/main/kotlin/com/github/kotlintelegrambot/dispatcher/handlers/NewChatMembersHandler.kt]: Done

File: telegram/src/main/kotlin/com/github/kotlintelegrambot/dispatcher/handlers/NewChatMembersHandler.kt [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+
+private class NewChatMembersHandlerProxy(
+    val handleNewChatMembers: HandleNewChatMembers
+) : HandleUpdate {
+    override fun invoke(bot: Bot, update: Update) {
+        val message = update.message
+        val newChatMembers = message?.newChatMembers
+        checkNotNull(message)
+        checkNotNull(newChatMembers)
+        handleNewChatMembers.invoke(bot, update.message, newChatMembers)
```
Comment (vjgarciag96) [inline telegram/src/main/kotlin/com/github/kotlintelegrambot/dispatcher/handlers/NewChatMembersHandler.kt:27]: Not really important, but you could use the local `message` val directly as argument for `invoke`. Also, I think it's enough to call `checkNotNull` just with `newChatMembers`; the compiler will infer that message is not null either.

Comment (imorte) [reply telegram/src/main/kotlin/com/github/kotlintelegrambot/dispatcher/handlers/NewChatMembersHandler.kt:27]: sure, i hurried

**Pull request merged**